### ISSUE TITLE
Syntax error with MySQL 8.0

### ISSUE THREAD

**Issue by vaficionado** [dschwen/g2piwigo#5]
Hi there - thanks for this great script. I'm migrating some long overdue G2 builds to Piwigo on modern infra and finding a rather inscrutable issue with MySQL 8.0 that I'm hoping you can help with.

Here's the output (sanitized)

```
UPDATE piwigo_images
  SET name=NULL
    , comment=NULL
    , date_available=''
  WHERE path = './galleries/index.php' 
UPDATE piwigo_categories SET name='NAME', comment='COMMENT', rank=8000, global_rank='8000' WHERE id = 1
DBD::mysql::st execute failed: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'rank=8000, global_rank='8000' WHERE id = 1' at line 1 at ./convertcomments.pl line 190, <STDIN> line 3.
```

I'm far from a Perl/MySQL person myself, so any help is much appreciated! I ran this successfully on an old box with MySQL 5.4 so I'm pretty confident it's not the structure of the G2 database or folder hierarchy, etc.

**Comment by dschwen** [comment]
I wonder if the quotes around 8000 in `global_rank='8000'` are the issue :-/

**Comment by vaficionado** [comment]
is that value coming from the database or is it a query that can be adjusted in the script somehow?

**Comment by dschwen** [comment]
Have you tried https://piwigo.org/ext/extension_view.php?eid=768 ?

**Comment by vaficionado** [comment]
Nice, I hadn't seen that - but it fails as well, with a totally different error that I can't troubleshoot "No Menalto tables found" (though they are definitely there, the connection string is correct, etc. Your locally executed perl does run, just with the originally noted error.

**Comment by vaficionado** [comment]
With some tweaking I was able to correct the behavior of the PHP extension, thanks for the tip. Appreciate the quick responses on such an old repo as well.

**Comment by dschwen** [comment]
@vaficionado could you summarize what you had to do to fix the PHP extension?

**Comment by vaficionado** [comment]
Oh, sure - I had to update line 140 in admin.php:

  `if (in_array($pt.'FileSystemEntity', $menalto_tables))`

Change `FileSystemEntity` to `filesystementity`

(pulled from https://piwigo.org/forum/viewtopic.php?id=24717)
